libs/live-fns/npm.js

```diff
@@ -33,25 +33,64 @@ module.exports = async function npm (method, ...args) {
 }
 
 async function pkg (topic, args) {
-  const endpoint = `https://unpkg.com/${args.join('/')}/package.json`
+  let pkg = args[0]
+  let tag = 'latest'
+  let isScoped = args[0].charAt(0) === '@'
+
+  if (isScoped) {
+    pkg = `${args[0]}/${args[1]}`
+    tag = args[2] || tag
+  } else {
+    pkg = `${args[0]}`
+    tag = args[1] || tag
+  }
+
+  const endpoint = `https://unpkg.com/${pkg}@${tag}/package.json`
   const meta = await axios.get(endpoint).then(res => res.data)
 
   switch (topic) {
-    case 'license':
+    case 'version': {
+      const color = tag !== 'latest'
+        ? 'cyan'
+        : (meta.version.split('.')[0] === '0' ? 'orange' : 'blue')
+
+      return {
+        subject: `npm${tag === 'latest' ? '' : '@' + tag}`,
+        status: `v${meta.version}`,
+        color
+      }
+    }
+    case 'license': {
       return {
         subject: 'license',
         status: meta.license || 'unknown',
         color: 'blue'
       }
-    case 'node':
+    }
+    case 'node': {
       return {
         subject: 'node',
         status: (meta.engines && meta.engines.node) || '*',
         color: 'green'
       }
+    }
+    default: {
+      return {
+        subject: 'npm',
+        status: 'unknown',
+        color: 'grey'
+      }
+    }
   }
 }
 
+pkg('version', ['ava'])
+pkg('version', ['ava', 'next'])
+pkg('version', ['next', 'canary'])
+pkg('version', ['babel-core'])
+pkg('version', ['@babel', 'core'])
+pkg('version', ['@nestjs', 'core', 'beta'])
```

Comment:
Is these for testing?

Reply:
Damn. I forgot them ;d But yea, that way i test the things :D